build.gradle

```diff
@@ -8,19 +8,23 @@ plugins {
 }
 
 group "com.mx.coppuccino"
-version "2.0.0"
+version "2.0.1"
```

Comment:
Let's the bump the version in a new PR to keep the version history visible in the commit history.